atlascope/core/models/waypoint.py

```diff
@@ -0,0 +1,10 @@
+from uuid import uuid4
+
+from django.contrib.gis.db import models as gisModels
+from django.db import models
+
+
+class Waypoint(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
```

Comment:
Same comment as above, re #120 and striking this line.

Reply:
resolved with a29845873608f22d5ba2d213129b54560c2522ae